### ISSUE TITLE
Job done <ID> [player]

### ISSUE THREAD

**Issue by LimeeFox** [StealWonders/EpicJobs#16]
would be nice if I could also pass in a player parameter, as somebody with extra privilege, to force-mark someone else's job as done (I accidentally reopened the wrong job and now I have to wait for the user to see it and mark it as done)

**Comment by StealWonders** [comment]
The `[player]` argument isn't required to implement this feature as far as I know. If a player isn't online it should still just set the job's status to done. Will add this in an upcoming version.

**Comment by StealWonders** [comment]
The current behavior for `/job done <id>` is that if you do not have it claimed then you cannot mark it done.
Do you want people with extra privileges to just override it automatically (always) or do you want to make sure the person knows what they are doing by having them enter `/job done <id> [<force=true>]` or like you suggested `/job done <id> [<player>]` 

Never the less you can still always use `/job edit status <id> done`

**Comment by LimeeFox** [comment]
Yeah /job done <id> should probably send a confirmation message warning that the job is claimed by someone else. And I didn't know about job edit status!

**Comment by StealWonders** [comment]
Implemented in cb11aa2 

The command is `/job done <id> --confirm`
